book/src/getting_started.md

```diff
@@ -25,17 +40,79 @@ Rust compiler. Here are the system requirements (they're pretty modest):
+$ cargo init --bin my_cool_game
+```
+Where my_cool_game is the name of your project.
+
+Next you'll want to add a dependency to amethyst.  To do this visit
+[https://crates.io/crates/amethyst] and grab the latest version number from that
+page.  Now go to the Cargo.toml file inside your project directory and add
+```toml
+[dependencies]
+amethyst = "x.x"
+```
+where x.x are the first two sections of the version number you grabbed from
+amethyst.  This way you can receive non-breaking patches from us with
+`cargo update` but you won't receive breaking changes unless you change your
+version number in Cargo.toml.
+This is because amethyst follows [semver](http://semver.org/spec/v2.0.0.html).
+
+## Amethyst specific setup
+
+Next you'll need a resources folder that will contain the data for your game.
+You can call this anything, but our convention is just `resources`.  Stick it in
+your project directory next to Cargo.toml.
+
+### config.ron
+
+  The first file you'll want to create in here is `config.ron` this file is
+written in the [RON](https://github.com/ron-rs/ron) format.  It's contents will
+look like this:
+
+```
+(
+  title: "My cool game",
+  dimensions: None,
+  max_dimensions: None,
+  min_dimensions: None,
+  fullscreen: false,
+  multisampling: 1,
+  visibility: true,
+  vsync: true,
+)
+```
+
+We'll go through each line.
+
+`title: "My cool game",` This sets the title of the game window.
+
+`dimensions: None,` This uses the Rust [Option](https://doc.rust-lang.org/std/option/enum.Option.html)
+type. This field if set will define the size of the window. Right now it's not
+ set. If we wanted to set it we'd use `Some((1920, 1080))` to get a 1920x1080
+ window.
+
+`max_dimensions: None,` This is similar to the dimensions field, but this sets a
+maximum size.
+
+`min_dimensions: None,` This is similar to the dimensions field, but this sets a
+minimum size.
+
+`fullscreen: false,` If this is true then this window will be fullscreen.
 
-1. [Use the Amethyst CLI tool to generate a new game project][as].
-2. [Create the Cargo crate and the "resources" folder structure yourself][ms].
+`multisampling: 1,` This defines the level of [MSAA anti-aliasing](https://en.wikipedia.org/wiki/Multisample_anti-aliasing).
 
-[as]: ./getting_started/automatic_setup.html
-[ms]: ./getting_started/manual_cargo_setup.html
+`visibility: true,` If this is false the window will not be immediately visible
+on startup.
 
-Since we're just getting started, it's fastest and highly recommended to use
-[Amethyst CLI][ac], which is included in the [amethyst_tools][at] crate. If
-you're of the intrepid type, you may go the vanilla Cargo route if you wish.
+`vsync: true,` If this is true then the game will use [vertical synchronization](https://en.wikipedia.org/wiki/Screen_tearing#V-sync).
 
-[ac]: https://github.com/amethyst/tools/tree/master/src/cli
-[at]: https://github.com/amethyst/tools
+This file is used to generate a `DisplayConfig` which we can then use to make a
```

Comment:
The grammar here seems odd. Maybe "be used to make"?

Reply:
Yeah that's much better.  Thanks!